ceph-releases/ALL/rhel8/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -0,0 +1,3 @@
+yum update -y && \
+yum install -y wget unzip util-linux python-setuptools udev device-mapper && \
```

Comment:
I'm testing this more. `python-setuptools` is not in RHEL 8, and it's `python3-setuptools`.